designs/noble/src/frontpoints.mjs

```diff
@@ -33,15 +92,13 @@ const createRightDartPoints = (points, Path, diff, utils) => {
     -0.5 * cbqc * points.armholeDartInside.dist(points.armholeDartTip)
   )
 
-  return new Path()
+  paths.princessSeam = new Path()
     .move(points.waistDartRight)
     .curve(points.waistCpUp, points.waistUpDartRightCpDown, points.waistUpDartRight)
-    .curve(
-      points.waistUpDartRightCpUp,
-      points.shoulderDartTipCpDownOutside,
-      points.shoulderDartOutside
-    )
-    .length()
+    .join(createTopRightDartPoints(points, Path, paths, options))
```

Comment:
You also need to remove the `paths` argument here (and maybe at locations where that method is called), or else it fails to run properly.